src/azure-cli-core/azure/cli/core/commands/__init__.py

```diff
@@ -322,22 +321,16 @@ def execute(self, args):
                 elif cmd.no_wait_param and getattr(expanded_arg, cmd.no_wait_param, False):
                     result = None
 
-                # TODO: Not sure how to make this actually work with the TRANSFORM event...
-                transform_op = cmd.command_kwargs.get('transform', None)
-                if transform_op:
-                    result = transform_op(result)
-
```

Comment:
@derekbekoe I tried to convert this to leverage EVENT_INVOKER_TRANSFORM_RESULT but I think it would potentially result in breaking changes.